### DISCUSSION TITLE
Release v1.0.2

### DISCUSSION THREAD

**Discussion by nitindt2** [ProjectiveGroupUK/tips-snowpark#6] [Announcements]
Now includes documentation, which can also be accessed from https://projectivegroupuk.github.io/tips-snowpark/

Also includes some minor fixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/ProjectiveGroupUK/tips-snowpark/releases/tag/v1.0.2'>Release v1.0.2</a>.</em>

**Comment by clockers1** [comment]
Nice work on the documentation
